### PR TITLE
PORTALS-1959: refactor old code to use new entity bundle hook

### PR DESCRIPTION
**Pull request by jay-hodgson**
define the dependencies necessary to update after oauth page reload

### CODE REVIEW

File: src/__tests__/lib/containers/DownloadConfirmation.test.tsx [hunk 1]
```diff
@@ -11,14 +11,13 @@ import {
...
-import { Alert } from 'react-bootstrap'
 import {
   MOCK_CONTEXT_VALUE,
   SynapseTestContext,
 } from '../../../mocks/MockSynapseContext'
 
-let getQueryTableResultsFn: Function
-let addFilesToDownloadRequestFn: Function
+let getQueryTableResultsFn: () => void
+let addFilesToDownloadRequestFn: () => void
```
Comment (jay-hodgson) [inline src/__tests__/lib/containers/DownloadConfirmation.test.tsx:20]: fix warning

File: src/lib/containers/download_list/DownloadConfirmation.tsx [hunk 2]
```diff
@@ -185,7 +186,7 @@ export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationP
   }, [accessToken])
 
   const {
-    data,
+    data: entityChildrenData,
```
Comment (jay-hodgson) [inline src/lib/containers/download_list/DownloadConfirmation.tsx:189]: renamed for clarity

File: src/lib/containers/download_list/DownloadConfirmation.tsx [hunk 3]
```diff
@@ -195,38 +196,30 @@ export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationP
...
-      ...queryOrFolder.queryBundleRequest!,
-      partMask,
-    }
-    const {queryCount, sumFileSizes} = await SynapseClient.getQueryTableResults(
-      queryBundleRequestSizeInformation,
-      token,
-    )
-    updateStats(queryCount, sumFileSizes?.sumFileSizesBytes)
-  }, [queryOrFolder, setStatus, updateStats])
+  const { data: queryResultBundle } = useGetQueryResultBundle(
```
Comment (jay-hodgson) [inline src/lib/containers/download_list/DownloadConfirmation.tsx:214]: Use the useGetQueryResultBundle hook instead

**Review by emmanmills — APPROVED** [review]
Looks good!

**Pull request merged**